### ISSUE TITLE
Error: The "path" argument must be of type string

### ISSUE THREAD

**Issue by findingorder** [bspaulding/rollup-plugin-image-files#2]
Tried including this in my rollup.config.js using
```JavaScript
import images from 'rollup-plugin-image-files'
```
and also included 'image()' in the plugins section.  Even without trying to import any image, it throws this error:
```
[!] TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string
TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string
    at assertPath (path.js:39:11)
    at Object.dirname (path.js:1331:5)
    at Object.ongenerate (/home/me/my-components/my-component/src/node_modules/rollup-plugin-image-files/dist/index.cjs.js:37:21)
    at /usr/local/lib/node_modules/rollup/dist/rollup.js:19615:40
    at Array.forEach (<anonymous>)
    at /usr/local/lib/node_modules/rollup/dist/rollup.js:19613:41
    at <anonymous>
```

**Comment by jmuenchenberg** [comment]
Encountered the same issue, with version 1.4.1

This happens only when using the code-splitting configuration for [rollup](https://rollupjs.org/guide/en/#code-splitting)

In my case this was fixed by changing line 14 in index.js to 
`		const dir = options.dir || path.dirname(options.dest || options.file);`

**Comment by bspaulding** [comment]
@jmuenchenberg thanks! fixed in 1.4.2 🤦‍♂
